mantidimaging/gui/windows/tomopy_recon/presenter.py

```diff
@@ -130,7 +132,7 @@ def _on_reconstruct_slice_done(self, task):
     def _on_reconstruct_volume_done(self, task):
         if task.was_successful():
             volume_data = task.result
-            volume_stack = Images(volume_data)
+            volume_stack = Images(volume_data, properties=self.stack_properties)
```

Comment:
I wonder if this could be a good PR to add recording the parameters of the reconstruction? Centres of rotation, projection angles (if provided), algorithm name and post-processing filter.

Do you think this can be done here or should it be moved to another PR?

Reply:
Yeah that'll be good to do - it seemed a bit weird to be ignoring it (and swapping axes) in the history.